src/test/java/de/retest/suite/flow/ApplyChangesToStatesFlowTest.java

```diff
@@ -0,0 +1,81 @@
+package de.retest.suite.flow;
+
+
+import static de.retest.Properties.WORK_DIRECTORY;
+import static de.retest.persistence.RecheckStateFileProviderImpl.RECHECK_PROJECT_ROOT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.HashSet;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.retest.Properties;
+import de.retest.configuration.Configuration;
+import de.retest.persistence.Persistence;
+import de.retest.persistence.PersistenceFactory;
+import de.retest.persistence.xml.util.StdXmlClassesProvider;
+import de.retest.ui.descriptors.Element;
+import de.retest.ui.descriptors.SutState;
+import de.retest.ui.diff.AttributeDifference;
+import de.retest.ui.review.ReviewResult;
+import de.retest.ui.review.SuiteChangeSet;
+import de.retest.util.junit.vintage.SystemProperty;
+
+public class ApplyChangesToStatesFlowTest {
+
+	private static final Logger logger = LoggerFactory.getLogger( ApplyChangesToStatesFlowTest.class );
+
+	private static final String SUITE = "ApplyChangesToStatesFlowTest";
+	private static final String TEST = "check_GUI_with_review_license";
+	private static final String ACTION = "launch";
+	private static final String FILE = "target/test-classes/recheck/" + SUITE + "/" + TEST + "." + ACTION + ".recheck";
```

Comment:
The file, respectively, workspace should point to a [temporary folder](https://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html).